edx_django_utils/monitoring/internal/backends.py

```diff
@@ -129,6 +156,9 @@ def record_exception(self):
         if span := self.dd_tracer.current_span():
             span.set_traceback()
 
+    def initialize_celery_monitoring(self, *args, **kwargs):
+        self.patch(celery=True)
```

Comment:
It looks like ddtrace patches Celery by default: https://github.com/DataDog/dd-trace-py/blob/01fbf9127180794392cc7dbe16acd610087dacf6/ddtrace/_monkey.py#L33 -- were you seeing otherwise?

Reply:
I was just going by these docs: https://ddtrace.readthedocs.io/en/stable/integrations.html#celery.
This was in the ticket I had created: https://github.com/edx/edx-arch-experiments/issues/584
Maybe it was inaccurate?

Separately: In those docs I see that trace headers are off by default. We should find out if that is what we want from DD? Maybe long running tasks would kill our traces? Not sure. @connorhaugh: Can you task/track this?